lms/djangoapps/django_comment_client/utils.py

```diff
@@ -134,9 +134,7 @@ def _sort_map_entries(category_map, sort_alpha):
     category_map["children"] = [x[0] for x in sorted(things, key=lambda x: x[1]["sort_key"])]
 
 
-def get_discussion_category_map(course):
-    course_id = course.id
-
+def get_discussion_category_map(course, ignore_always_cohort_inline_discussions=False):
```

Comment:
Can you add a docstring explaining the purpose of the parameters?

Also I am thinking, when fetching the map for the instructor dashboard besides ignoring the `always_cohort_inline_dicussions` setting, shouldn't the `is_cohorted` setting also be ignored? Or in other words a topic should only be marked `is_cohorted` if it is in the list.

If yes, you can call the parameter something like `cohorted_if_in_list`.